.buildkite/pipeline.yml

```diff
@@ -252,7 +252,19 @@
       --test_env=RAY_USE_MULTIPROCESSING_CPU_COUNT=1
       rllib/...
 
-- label: ":brain: RLlib: everything else"
+- label: ":brain: RLlib: Trainer Tests"
```

Comment:
What is the intended change here?

Reply:
To separate the trainer compilation tests (basically all tests that are in the `agents` dir) from the other tests in this suite (e.g. models, policy, utils, etc..).